src/osd/OSD.cc

```diff
@@ -6655,6 +6655,11 @@ bool OSD::ms_get_authorizer(int dest_type, AuthAuthorizer **authorizer, bool for
 {
   dout(10) << "OSD::ms_get_authorizer type=" << ceph_entity_type_name(dest_type) << dendl;
 
+  if (is_stopping()) {
+    dout(10) << __func__ << " bailing, we are shutting down" << dendl;
```

Comment:
@badone maybe we could make it a `derr()` or even an `assert()` ? with the fix of `session->con->mark_down();`, do you think we can run into this method if OSD is shutting down?